Makefile

```diff
@@ -29,8 +29,8 @@ STATIC := $(STATIC)
 # depending on the context.
 COCKROACH_IMAGE :=
 
-RUN  := run
-GOPATH  := $(CURDIR)/_vendor:$(CURDIR)/../../../..
+RUN := run
+export GOPATH := $(CURDIR)/_vendor:$(CURDIR)/../../../..
```

Comment:
So if the variable exists in the environment and is changed in make, make will re-export it? Weird.

I don't set GOPATH globally, but my emacs setup detects when I'm in a go project and does the equivalent of `export GOPATH=$(make gopath)`. When this broke, I was running without a GOPATH and was surprised to see that `make test` no longer worked.